src/data/set/lattice.lean

```diff
@@ -1584,23 +1584,36 @@ lemma disjoint_image_of_injective {f : α → β} (hf : injective f) {s t : set
   (hd : disjoint s t) : disjoint (f '' s) (f '' t) :=
 disjoint_image_image $ λ x hx y hy, hf.ne $ λ H, set.disjoint_iff.1 hd ⟨hx, H.symm ▸ hy⟩
 
-lemma disjoint_preimage {s t : set β} (hd : disjoint s t) (f : α → β) :
```

Comment:
In case anyone else was confused like I was, this lemma isn't gone, it was a duplicate of `disjoint.preimage`

Reply:
@YaelDillies Perhaps you could edit the PR description to reflect this? The lemma you mention in the title was already in mathlib, and you can instead talk about what's actually new

Reply:
Done!